docs/pull_request_template.md

```diff
@@ -20,7 +20,7 @@ Fixes # (PUT ISSUE NUMBER AFTER #)
   - If you added config, describe how we use this tool in the config’s comment.
   - If you added something to the project’s architecture, describe it in the project’s `README.md`.
   - Try to focus on “why?”, not “how?”.
-- [ ] If you added a new dependency, check our [requirements](../README.md#dependencies).
+- [ ] If you added a new dependency, check our [requirements](../blob/main/README.md#dependencies).
```

Comment:
Try a link in your fork https://github.com/pshergie/slowreader/blob/fix-pr-body-checklist-link/docs/pull_request_template.md

It doesn’t work right now (404).

What exact issue do you have right now?

Reply:
the issue: the `requirements` link doesn't work from PRs

Reply:
In this case try to avoid `../` in the link, give a direct one `https://`

Reply:
looks like `[a relative link](other_file.md)` could work https://github.blog/2013-01-31-relative-links-in-markup-files/

Reply:
Sometimes docs are old. That one from 2013.

You see that link doesn’t work if you open `.md` file in GitHub.

Reply:
@ai ready for the review 🙇